### ISSUE TITLE
Use this frontend with auth server.

### ISSUE THREAD

**Issue by chendoubao** [kwk/docker-registry-frontend#77]
HI,
    First,thanks very much for this frontend.
    In my family , i use registry v2 with an auth server(http://the.binbashtheory.com/creating-private-docker-registry-2-0-with-token-authentication-service/) which use token. So i think this frontend have some problems with token auth registry. Do you think so ?

**Comment by kwk** [comment]
Right. The frontend currently can handle Kerberos authentication and has no notion of authorization. It is planned though. I have an out of the box registry setup with token authorization and LDAP backend here if you'd like to work with it: https://github.com/kwk/docker-registry-setup

**Comment by chendoubao** [comment]
Thank you by this.
I hope your frontend can use with the auth server . That is cool . 
Do you have time plan for this ?

**Comment by kwk** [comment]
No

**Comment by chendoubao** [comment]
Oh.., that's all right....

**Comment by yangtao309** [comment]
@chendoubao +1

**Comment by djenriquez** [comment]
+1, looks to be one of the few viable front-ends available. There is gigantic opportunity for this project to be an essential part of building a private registry if it is able to handle the security requirements that many have. That would mean being able to support the configurations that [Docker-Distribution](https://github.com/docker/distribution) have available.

I'm not an expert with Angular, but i'm sure someone out there has time to help get this project caught up. :grin:

**Comment by srikarm** [comment]
+1

@kwk, would you be able to write an approach you would take for this to work with auth server? someone may attempt to code based on that...

**Comment by cromat** [comment]
@chendoubao +1
This would be great feature.

**Comment by castorls** [comment]
+1 for me also

**Comment by electrical** [comment]
I deployed this front-end at work and just started to implement the authentication and indeed the web interface fails now. Would love to have it working with the authentication.

Only other way is to setup a second registry service without authentication specifically for the front-end using the same data directory.

**Comment by yuxiaokui** [comment]
这拒绝的也太决绝了。  惨烈。。   蓝瘦香菇。

**Comment by yuxiaokui** [comment]
I have an idea,you can start a new registry with out auth-server.  And you can use this web to view the docker images in the registry.

**Comment by robermorales** [comment]
any update on supporting more auth methods?

thanks in advance
